tests/test_add_port.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from freezeyt.util import parse_absolute_url, add_port
+
+url_with_port = parse_absolute_url('http://localhost:80/')
+
+def test_add_http_port():
+    url_without_port = url_with_port.join('http://localhost/')
```

Comment:
Why we don't use just `parse_absolute_url('http://localhost/')`

Reply:
Our `parse_absolute_url` always adds the port :)